### PR TITLE
created an api endpoint to fetch all the standup details from database

### PR DESCRIPTION
**Pull request by ayushdevelop**
# Related Issue

-  https://github.com/code-mancers/g-bot/issues/18

# functionalities Added or Changed

- created an endpoint in the application backend which will help frontend to fetch standup data from database
- on a request send to server to fetch standup data will initiate the process of extraction of standup data 
- the `Prisma` client will interact with `Postgres` database on receiving a request to fetch standup data

# Checklist

- [x] Tests
- [x] Uploaded screenshots

# Screenshots

- Standups.service.test
Screenshot 2022-10-18 at 11 12 44 AM

- Standups.controller.test
Screenshot 2022-10-18 at 8 46 31 AM

### CODE REVIEW

File: server/src/__tests__/standup/standups.controller.test.ts [hunk 1]
```diff
@@ -26,3 +27,18 @@ describe("Standups", () => {
...
   });
 });
+
+const mockGetStandups = getAllStandups as jest.MockedFunction<
+  typeof getAllStandups
+>;
+
+describe("Standups", () => {
+  describe("GET /standups", () => {
+    test("should return the title of standup as random", async () => {
```
Comment (SujayPrabhu96) [inline server/src/__tests__/standup/standups.controller.test.ts:37]: Also add assertions for standup questions and members and channel

File: server/src/services/standups.service.ts [hunk 2]
```diff
@@ -54,3 +54,11 @@ export const postStandup = async ({
     throw error;
   }
 };
+
+export const getAllStandups = async () => {
+  const standups = await prisma.standup.findMany({
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:59]: Add error handling please

File: server/src/__tests__/standup/standups.service.test.ts [hunk 3]
```diff
@@ -65,3 +81,11 @@ describe("Create Standup", () => {
     );
   });
 });
+
+// test for fetching standup
+describe("Fetch Standup", () => {
+  test("it should fetch all the standups from database", async () => {
```
Comment (SujayPrabhu96) [inline server/src/__tests__/standup/standups.service.test.ts:87]: test for failure cases also

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/controllers/standup.controller.ts [hunk 4]
```diff
@@ -9,3 +9,12 @@ export const postStandupHandler = async (req: Request, res: Response) => {
     res.status(500).json({ message: error });
   }
 };
+
+export const fetchStandupHandler = async (req: Request, res: Response) => {
+  try {
+    const standups = await getAllStandups();
+    res.status(200).set("Content-Range", `${standups.length}`).json(standups);
```
Comment (SujayPrabhu96) [inline server/src/controllers/standup.controller.ts:16]:
```suggestion
    res.status(200).set("Content-Range", standups.length).json(standups);
```

File: server/src/index.ts [hunk 5]
```diff
@@ -9,7 +9,7 @@ handleListeners(slackApp);
 
 standupScheduled(slackApp);
 
-expressApp.listen(process.env.PORT || 8000, () => {
+expressApp?.listen(process.env.PORT || 8000, () => {
```
Comment (SujayPrabhu96) [inline server/src/index.ts:12]:
```suggestion
expressApp.listen(process.env.PORT || 8000, () => {
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/controllers/standup.controller.ts (hunk 4 above)
Comment (ayushdevelop) [reply server/src/controllers/standup.controller.ts:16]: here we need to pass a strjng instead of a number, that is why I wrote it that way

Comment (SujayPrabhu96) [reply server/src/controllers/standup.controller.ts:16]: Okay, in that case you can do `standups.length.toString()`

**Review by SujayPrabhu96 — APPROVED** [review]

**Pull request merged**